cluster/rackspace/README.md

```diff
@@ -0,0 +1,5 @@
+# Rackspace
```

Comment:
I don't think that this file is necessary. We are trying to clear out the whole cluster directory, so the absence of a rackspace subdirectory won't seem strange.